src/SplitPackages/project.json

```diff
@@ -16,6 +16,11 @@
         "NuGet.Client": "3.4.0-*"
     },
     "frameworks": {
+        "dnxcore50": {
+            "dependencies": {
+                "NETStandard.Library": "1.0.0-rc2-*"
```

Comment:
@shirhatti Whatever you prefer, we filter our packages to RC2, so the end result is going to be the same